test/e2e/resources/xgboost_model_package_group.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: ModelPackageGroup
+metadata:
+  name: $MODEL_PACKAGE_GROUP_NAME
+spec:
+  modelPackageGroupName: $MODEL_PACKAGE_GROUP_NAME
```

Comment:
interesting, group can be created without any packages in it

Reply:
Yes group can be created without packages. Can be created without packages in boto api aswell as sagemaker ui.